resources/public/api/conf/2.0/examples/retrieveBsas/retrieve_foreign_property_fhl_eea_bsas.json

```diff
@@ -0,0 +1,55 @@
+{
+  "metadata": {
+    "typeOfBusiness": "foreign-property-fhl-eea",
+    "accountingPeriod": {
+      "startDate": "2021-04-06",
+      "endDate": "2022-04-06"
+    },
+    "taxYear": "2021-22",
+    "requestedDateTime": "2022-10-14T11:33:27Z",
+    "bsasId": "f2fb30e5-4ab6-4a29-b3c1-c7264259ff1c",
+    "summaryStatus": "valid",
+    "adjustedSummary": true
+  },
+  "bsas": {
+    "total": {
+      "income": 123.12,
+      "expenses": 123.12,
+      "additions": 123.12,
+      "deductions": 123.12
+    },
+    "profit": {
+      "net": 123.12,
+      "taxable": 123.12
+    },
+    "loss": {
+      "net": 123.12,
+      "adjustedIncomeTax": 123.12
+    },
+    "incomeBreakdown": {
+      "rentIncome": 123.12
+    },
+    "expensesBreakdown": {
+      "premisesRunningCosts": 123.12,
+      "repairsAndMaintenance": 123.12,
+      "financialCosts": 123.12,
+      "professionalFees": 123.12,
+      "travelCosts": 123.12,
+      "costOfServices": 123.12,
+      "other": 123.12,
+      "consolidatedExpenses": 123.12
```

Comment:
consolidatedExpenses shouldn't be included with other things, i think just removing it would be fine or alternatively have another example for consolidated expenses